### PR TITLE
Fixed Errors on api Suite Test

### PR DESCRIPTION
**Pull request by chay22**
Hello, this PR is made to fix all thrown errors on `api` suite test. My former intention was to fix the `/users` page due to:

- Can not create, delete, update, basically anything because of syntax error
- The `Show Deleted Users` always empty because of no (more) `scopeGetDeleted` scope on model which caused a thrown error.

### CODE REVIEW

**Comment by cahyadinumedia** [comment]
Whoops, I pushed too many changes. I don't think this should be merged.

**Comment by snipe** [comment]
Thanks! Is this ready for review?

**Comment by chay22** [comment]
> Thanks! Is this ready for review?

Yes, please. This bloated already.

File: app/Http/Requests/SaveGroupRequest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|min:3',
```
Comment (snipe) [inline app/Http/Requests/SaveGroupRequest.php:27]: Wouldn't this make more sense in the model level validation, since that's all it's doing here?

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit after asking for clarification/changes, so I'm going to close this PR for now, but feel free to grab the latest off of develop and see the issue remains.

hackerman-hacking

**Pull request closed without merging**